internal/generate/testdata/go/pkg/apis/cache/v1alpha1/dummy_types.go

```diff
@@ -22,8 +22,8 @@ import (
 // +k8s:openapi-gen=false
 type NoKindSpec struct {
 	// Not included in anything, no kind type
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
-	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=status
```

Comment:
It shows a breaking change where the annotations changed. So, we would need to inform I in the changelog and migration. Also, I think we need to make it clear in the first comment of this PR.

Reply:
I need to update several docs, just waiting for discussion of changes to marker format beforehand.

Reply:
Done.